style/checkstyle.xml

```diff
@@ -282,7 +282,6 @@
 
         <!-- Checks for out of order import statements. -->
         <module name="ImportOrder">
-            <property name="severity" value="info"/>
```

Comment:
@lamber-ken shall we keep this config option?

Reply:
All finished.

Reply:
OK, since this is the last shot, we can remove it(let it keep the default value **error**). Ever since this time point, all the classes must respect this import order rule, otherwise, the checkstyle will break the build progress. cc @vinothchandar